Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -662,15 +662,19 @@ class FastStringifier {
     String firstGetterSetterPropertyName() const;
     void recordFastPropertyEnumerationFailure(JSObject&);
     bool haveFailure() const;
-    unsigned remainingCapacity() const;
+    bool hasRemainingCapacity(unsigned size = 1);
+    bool hasRemainingCapacitySlow(unsigned size);
     bool mayHaveToJSON(JSObject&) const;
 
     static void logOutcome(ASCIILiteral);
     static void logOutcome(String&&);
 
+    static unsigned useableBufferSize(size_t availableBufferSize);
```

Comment:
This has a spelling error or typo in the word _usable_. Repeated throughout.

Given that the argument is the number of bytes available in something that has a maximum of 6000, it seems to me that passing a `size_t` is overkill, it would even fit in `uint6_t`.

Reply:
I changed "useable" to "usable", and this `size_t` to `unsigned`.  I'm not sure there's much to be gained from using a smaller size.  The codegen on ARM64 might even be slightly less efficient if we go smaller.